src/java.desktop/share/classes/sun/awt/FontConfiguration.java

```diff
@@ -837,9 +837,9 @@ private void reorderSequenceForLocale(String[] seq) {
         }
     }
 
-    private static Vector<String> splitSequence(String sequence) {
-        //String.split would be more convenient, but incurs big performance penalty
-        Vector<String> parts = new Vector<>();
+    private static String[] splitSequence(String sequence) {
+        //String.split would be more convenient, but it behaves slightly different
+        ArrayList<String> parts = new ArrayList<>();
```

Comment:
Did you check that the difference in handling empty strings is important here? Do the code below actually use it so we cannot reuse the "String.split()"?

Reply:
Result of splitSequence is used to fill `scriptIDs` map

https://github.com/openjdk/jdk/blob/4be94e435002d9d6cb594a393e9e35d650a9a0c9/src/java.desktop/share/classes/sun/awt/FontConfiguration.java#L2118-L2132

So, for example for `",,"` String:
* `splitSequence` returns 2 empty elements as result. And `scriptIDs` will contain the key `""` after that
* `String.split` returns no elements. And `scriptIDs` will not contain the key `""` after that

Not sure, how important it is, but seems it could amend behavior of the program.